package.json

```diff
@@ -27,7 +27,8 @@
   ],
   "scripts": {
     "test": "npx gulp test --force --speak",
-    "testall": "npx gulp test --browsers all --force --speak",
+    "testall": "npm run test:assembler && npx gulp test --browsers all --force --speak",
+    "test:assembler": "npx mocha tools/assembler/test/test-main.js",
```

Comment:
Why did you renamed `index.js` to `test-main.js`?

Reply:
Renamed back to `index.js`.